b2/console_tool.py

```diff
@@ -1123,8 +1123,7 @@ class TestRawApi(Command):
     PRIVATE = True
 
     def run(self, args):
-        test_raw_api()
-        return 0
```

Comment:
ouch

Reply:
should raise an exception though, it's the shell script that hid the error, right?

Reply:
Command classes in `console_tool.py` return 0 or non-zero status.  And `test_raw_api` follows that convention.

I would be fine with changing `test_raw_api` to raise an exception if it sees a problem.